src/renderers/dom/fiber/__tests__/ReactDOMFiber-test.js

```diff
@@ -44,6 +44,28 @@ describe('ReactDOMFiber', () => {
     expect(container.textContent).toEqual('10');
   });
 
+  it('should render dangerouslySetInnerHTML', () => {
```

Comment:
Thank you for moving the test. Can you remove the ones in this file now? We don't need duplicates because `ReactDOMComponent-test` is run in both Stack and Fiber mode.